### PR TITLE
Camera: Add VM configuration script

### PR DESCRIPTION
**Pull request by kantoniak**
Add script to set up VirtualBox VM with Raspbian. No webcam or USB support as of now.

### CODE REVIEW

File: camera/set-up-vm.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# TODO: Force running from this dir
+CURRENT_DIR="./"
+
+PROJECT_NAME=ivory-beam
+
+RASPBIAN_ISO_URL="https://downloads.raspberrypi.org/rpd_x86/images/rpd_x86-2020-02-14/2020-02-12-rpd-x86-buster.iso"
+RASPBIAN_BUILD="2020-02-12-rpd-x86-buster"
+
+MACHINE_NAME="${PROJECT_NAME}-camera"
+MACHINE_DIR="machine-files"
```
Comment (avan1235) [inline camera/set-up-vm.sh]: add option to set the install location

File: camera/set-up-vm.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# TODO: Force running from this dir
```
Comment (avan1235) [inline camera/set-up-vm.sh]: TODO

File: camera/set-up-vm.sh (hunk 1 above)
Comment (kantoniak) [reply camera/set-up-vm.sh]: Done.

File: camera/set-up-vm.sh (hunk 2 above)
Comment (kantoniak) [reply camera/set-up-vm.sh]: Done.

**Comment by kantoniak** [comment]
@avan1235: It would be great if you could go through the process on your own.

File: camera/set-up-vm.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+mkdir -p "${VM_DIR}"
+echo "Using vm dir: ${VM_DIR}"
+
+echo "Downloading image..."
+wget -q --show-progress "${RASPBIAN_ISO_URL}" -O "${VM_ISO_PATH}"
+
+echo "Creating vm..."
+vboxmanage createvm --name "${VM_NAME}" --ostype "Linux" --register --basefolder "${CURRENT_DIR}" && # Linux OS type is "Other Linux (32-bit)"
+vboxmanage modifyvm "${VM_NAME}" --memory 6144 --vram 128 && # 6 GB RAM, 128 MB VRAM
```
Comment (avan1235) [inline camera/set-up-vm.sh]: We cannot use th VM with so much RAM as it is not practical to take so much resources from host computer + raspberry has 2 GBs of RAM so I don't see any reason to give so much memory to VM (if we want fast compilation/better performance on tests we can test it on local machines with opencv)

Comment (kantoniak) [reply camera/set-up-vm.sh]: I missed differences in host systems. c07f7e6 comes with extra flags for RAM and disk size. Also, RAM default is now 512 MB to match Raspberry Pi 3 A+ ([official spec](https://www.raspberrypi.org/products/raspberry-pi-3-model-a-plus/)).

Scripts are needed nevertheless, you can never guarantee your environment will exactly match one on the device. Trivial example: if somebody uses non-debian derivative they can't run `apt-get` commands for updating Pi libraries, not speaking of version mismatches and so on...

**Pull request merged**